src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './Navbar.scss';
+
+export const Navbar: React.FC = () => (
+  <nav className="nav">
+    <div className="logo">
+      <Link to="/">buk klab</Link>
+    </div>
+    <ul>
+      <li>
+        <Link to="/books">books</Link>
+      </li>
+      <li>
+        <Link to="/members">members</Link>
+      </li>
+      <li>
+        <Link to="/about">about</Link>
+      </li>
+      <li>
+        <Link to="/signin">sign in</Link>
+      </li>
+      <button type="button" className="button">
+        join buk klab
```

Comment:
This doesnt work in safari

<img width="202" alt="CleanShot 2024-11-05 at 21 45 08" src="https://github.com/user-attachments/assets/1680f110-4053-46a6-81be-d68f83de30b2">


Yes, I have tested it even there :D